ui/core/components/suggest_reforges_action.tsx

```diff
@@ -1001,4 +1003,62 @@ export class ReforgeOptimizer {
 		}
 		return statPoints;
 	}
+
+	onReforgeDone() {
+		const itemSlots = this.player.getGear().getItemSlots();
+		const changedSlots: (ReforgeData | undefined)[] = Array(itemSlots.length);
```

Comment:
If you were to do the above, then `changedSlots` could be another `Map<ItemSlot, ReforgeStat>`, with a very simple loop to just compare the two maps for summarizing the changes.